ignite/distributed/comp_models/native.py

```diff
@@ -371,8 +371,14 @@ def spawn(  # type: ignore[override]
 
             start_processes = mp.spawn
             # start_method and start_processes in pytorch >= 1.5
+            default_start_method = "spawn"
```

Comment:
If the parameter `default_start_method` is useful only when `torch.__version__ >= 1.5.0`, this line should be moved to L376.